docs/integrations/data-ingestion/clickpipes/mongodb/index.md

```diff
@@ -0,0 +1,108 @@
+---
+sidebar_label: 'Ingesting Data from MongoDB to ClickHouse'
+description: 'Describes how to seamlessly connect your MongoDB to ClickHouse Cloud.'
+slug: /integrations/clickpipes/mongodb
+title: 'Ingesting data from MongoDB to ClickHouse (using CDC)'
+---
+
+import BetaBadge from '@theme/badges/BetaBadge';
+import cp_service from '@site/static/images/integrations/data-ingestion/clickpipes/cp_service.png';
+import cp_step0 from '@site/static/images/integrations/data-ingestion/clickpipes/cp_step0.png';
+import mongodb_tile from '@site/static/images/integrations/data-ingestion/clickpipes/mongodb/mongodb-tile.png'
+import mongodb_connection_details from '@site/static/images/integrations/data-ingestion/clickpipes/mongodb/mongodb-connection-details.png'
+import ssh_tunnel from '@site/static/images/integrations/data-ingestion/clickpipes/postgres/ssh-tunnel.jpg'
+import select_destination_db from '@site/static/images/integrations/data-ingestion/clickpipes/mongodb/select-destination-db.png'
+import ch_permissions from '@site/static/images/integrations/data-ingestion/clickpipes/postgres/ch-permissions.jpg'
+import Image from '@theme/IdealImage';
+
+# Ingesting data from MongoDB to ClickHouse (using CDC)
+
+<BetaBadge/>
+
+:::info
+Currently, ingesting data from MongoDB to ClickHouse Cloud via ClickPipes is in Private Preview.
+:::
+
+You can use ClickPipes to ingest data from your MongoDB database into ClickHouse Cloud. The source MongoDB database can be hosted on-premises or in the cloud using services like Mongo Atlas.
```

Comment:
“Mongo Atlas” should read MongoDB Atlas throughout; the generic guide gets this right but the main index.md and prerequisite links don’t.

Reply:
good catch!